model_analyzer/config/generate/optuna_run_config_generator.py

```diff
@@ -44,20 +51,31 @@
 
 logger = logging.getLogger(LOGGER_NAME)
 
+TrialObjectives: TypeAlias = Dict[str, Any]
+ParameterCombo: TypeAlias = Dict[str, Any]
+
 
 class OptunaRunConfigGenerator(ConfigGeneratorInterface):
     """
     Use Optuna algorithm to create RunConfigs
     """
 
+    # This list represents all possible parameters Optuna can currently search for
+    optuna_parameter_list = [
+        "batch_sizes",
+        "instance_group",
+        "concurrency",
+        "max_queue_delay_microseconds",
+    ]
+
     def __init__(
         self,
         config: ConfigCommandProfile,
         gpu_count: int,
         models: List[ModelProfileSpec],
         model_variant_name_manager: ModelVariantNameManager,
-        search_parameters: SearchParameters,
-        seed: Optional[int] = 0,
+        search_parameters: Dict[str, SearchParameters],
```

Comment:
Why dict? Is it for multi-model support?

Reply:
Correct. Also needed for ensemble/BLS/LLM